### PR TITLE
RIA-3377: openid changes

### PR DESCRIPTION
**Pull request by jkumar-moj**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RIA-3377

### Change description ###
OpenID changes

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/infrastructure/security/idam/IdamUserDetailsProviderTestOld.java [hunk 1]
```diff
@@ -0,0 +1,306 @@
+//package uk.gov.hmcts.reform.iahomeofficeintegrationapi.infrastructure.security.idam;
```
Comment (makowalski) [inline src/test/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/infrastructure/security/idam/IdamUserDetailsProviderTestOld.java:1]: I think we do not need this file

Comment (jkumar-moj) [reply src/test/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/infrastructure/security/idam/IdamUserDetailsProviderTestOld.java]: Thanks Marcin. The file is now removed.

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Comment by jkumar-moj** [comment]
Reopening as it was automatically closed due to inactivity

**Pull request merged**